astromodels/functions/functions.py

```diff
@@ -337,6 +337,57 @@ def evaluate(self, x, K, piv, index, xc):
 
         return K * np.exp(log_v)
 
+class inverse_cutoff(Function1D):
```

Comment:
Let's try to keep the naming scheme similar to the other functions. These are classes, so it should be capitalized. Also, the name is not so informative. Perhaps expand upon the name to help users?